package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "use-http",
-  "version": "0.1.22",
-  "main": "src/main.js",
+  "version": "0.1.23",
+  "main": "build/main.js",
+  "peerDependencies": {
+    "react": "^16.8.0"
+  },
   "dependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
     "react-scripts": "2.1.8",
     "react-usefetcher": "^0.1.12"
   },
+  "devDependencies": {
+    "react-dom": "^16.8.6"
```

Comment:
react-dom is only really used by the unshipped App.js, so moving this to dev. deps.